### PR TITLE
A couple of message logs fixes

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 1a4d01e1e3061eb7fbc4976b4730564d43c3da20:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-1071e3a7b61e0decca5ac4ecc7ea67290f17b50c-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-1071e3a7b61e0decca5ac4ecc7ea67290f17b50c-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-1071e3a7b61e0decca5ac4ecc7ea67290f17b50c-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-1071e3a7b61e0decca5ac4ecc7ea67290f17b50c-x86.apk)

**Comment by xBcai** [comment]
Can the title of this be centered?
IMG_20210302_135334

File: src/qml/qgismobileapp.qml [hunk 1]
```diff
@@ -1332,7 +1332,7 @@ ApplicationWindow {
     }
 
     MenuItem {
-      text: qsTr( "View Log Messages" )
+      text: qsTr( "Messages Logs" )
```
Comment (m-kuhn) [inline src/qml/qgismobileapp.qml:1335]: Compare with "Message Log" in the toasts. Keep it singular?

Comment (xBcai) [reply src/qml/qgismobileapp.qml:1335]: Now this is displayed correctly

File: src/qml/qgismobileapp.qml [hunk 2]
```diff
@@ -1332,7 +1332,7 @@ ApplicationWindow {
     }
 
     MenuItem {
-      text: qsTr( "Messages Logs" )
+      text: qsTr( "Messages Log" )
```
Comment (m-kuhn) [inline src/qml/qgismobileapp.qml:1335]:
```suggestion
      text: qsTr( "Message Log" )
```

Shouldn't it be "message log" along the line of "task bar", "menu bar"?

**Comment by nirvn** [comment]
@m-kuhn , review addressed.

**Review by m-kuhn — APPROVED** [review]
Thanks !

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 5b60e5f2d3c6b27b6a733f650f88f8e30a1f8e5e:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-4d3d2cf39f9279e8b254211c1dff8528f71ec7c2-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-4d3d2cf39f9279e8b254211c1dff8528f71ec7c2-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-4d3d2cf39f9279e8b254211c1dff8528f71ec7c2-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-4d3d2cf39f9279e8b254211c1dff8528f71ec7c2-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 15c5fc6b0637a8f07d20f9154cebb9d241a24345:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-de16ae1388c97eba5d858aa1c91d6dd6679aa82b-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-de16ae1388c97eba5d858aa1c91d6dd6679aa82b-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-de16ae1388c97eba5d858aa1c91d6dd6679aa82b-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1734-de16ae1388c97eba5d858aa1c91d6dd6679aa82b-x86.apk)

**Pull request merged**